3.6.0/postgres/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM geonetwork:3.6.0
+
+RUN apt-get update && apt-get install -y postgresql-client && \
+    rm -rf /var/lib/apt/lists/*
+
+#Set PostgreSQL as default GN DB
+RUN sed -i -e 's#<import resource="../config-db/h2.xml"/>#<!--<import resource="../config-db/h2.xml"/> -->#g' $CATALINA_HOME/webapps/geonetwork/WEB-INF/config-node/srv.xml && \
+sed -i -e 's#<!--<import resource="../config-db/postgres.xml"/>-->#<import resource="../config-db/postgres.xml"/>#g' $CATALINA_HOME/webapps/geonetwork/WEB-INF/config-node/srv.xml
+
+COPY ./jdbc.properties $CATALINA_HOME/webapps/geonetwork/WEB-INF/config-db/jdbc.properties
+
+#Initializing database & connection string for GN
+COPY ./docker-entrypoint.sh /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
+
+CMD ["catalina.sh", "run"]
```

Comment:
@gannebamm I get some errors while running this image. Not sure if it the bug is on the support to postgres in GN, itself. @juanluisrp ?

Reply:
I'm checking it today, @doublebyte1.

Reply:
@juanluisrp  The problem should be solved with [this](https://github.com/geonetwork/docker-geonetwork/commit/27b11755dc0fa7fcf5be261c089c4b359a4be304) commit.